templates/openwisp2/settings.py

```diff
@@ -81,6 +82,7 @@
 ]
 
 MIDDLEWARE_CLASSES = [
+    'django.middleware.common.CommonMiddleware',
```

Comment:
hey, this this already present, why are you adding it here?

Reply:
Oh, didn't notice.